test/ldapsync.py

```diff
@@ -104,6 +104,26 @@ def test_sync_altered_accounts(self, mock_add_or_update):
         ldap_attrs = {'status': ['active'], 'scratchDirectory': ['/user/scratch/gent/vsc400/vsc40075'], 'dataDirectory': ['/user/data/gent/vsc400/vsc40075'], 'cn': 'vsc40075', 'homeQuota': ['5242880'], 'institute': ['gent'], 'loginShell': ['/bin/bash'], 'uidNumber': ['2540075'], 'researchField': ['Bollocks'], 'gidNumber': ['2540075'], 'gecos': ['Foo Bar'], 'dataQuota': ['1'], 'homeDirectory': ['/user/home/gent/vsc400/vsc40075'], 'mail': ['foobar@ugent.be'], 'scratchQuota': ['1'], 'pubkey': ['pubkey1', 'pubkey2'], 'instituteLogin': ['foobar'], 'uid': ['vsc40075']}
         mock_add_or_update.assert_called_with(VscLdapUser, test_account.vsc_id, ldap_attrs, True)
 
+
+    @skipIf(is_py3(), "Python 2 only")
```

Comment:
why is there no py3 test?

Reply:
Because we may assume that ensure_ascii_string is tested in vsc-install and that this will not affect the code flow or its operation?

Reply:
Adding a test for py3 will only test that this specific function also works in py3, which is not something we need to do here?

Reply:
then what are you testing in py2? or what is different in py2 vs py3?
i'm confused why this exception is needed at all

Reply:
I am testing that the unicode stuff is replaced as expected.

Reply:
so that should be tested on python3 and python2 no? I know the order differs (I had that on other repos as well) . But either make a test that works on both, or make 2 tests.